frontend/components/product-list/sections/ProductListChildrenSection.tsx

```diff
@@ -142,7 +127,7 @@ export function ProductListChildrenSection({
 }
 
 interface ChildLinkProps {
-   child: ProductListChild;
+   child: ProductList['children'][0];
```

Comment:
I didn't know you could do this, cool! I'm curious if there is any advantage to this syntax as opposed to explicitly listing `ProductListChild` as we did previously?

https://www.typescriptlang.org/docs/handbook/2/indexed-access-types.html

Reply:
`ProductListChild` was a locally defined type. I dropped it in favour of using the type that comes from the model, just for consistency and to avoid having to maintain and update a separate type